src/txHashBuilder.c

```diff
@@ -1184,6 +1204,9 @@ static void txHashBuilder_assertCanLeaveMint(tx_hash_builder_t* builder)
 	default:
 		ASSERT(false);
 	}
+
+	ASSERT(builder->multiassetData.remainingAssetGroups == 0);
+	ASSERT(builder->multiassetData.remainingTokens == 0);
```

Comment:
This should not be moved?

Reply:
That's interesting. The moving back was part of the commit chain, but should have been resolved and dissolved when git encountered the rebase to the moved back (to the multisig branch) move of the asserts.
Done.